tools/check-help.sh

```diff
@@ -1,41 +1,35 @@
-#!/usr/bin/env bash
+#!/bin/sh -eu
 # SPDX-License-Identifier: LGPL-2.1-or-later
-set -eu
-set -o pipefail
-
-# Note: `grep ... >/dev/null` instead of just `grep -q` is used intentionally
-#       here, since `grep -q` exits on the first match causing SIGPIPE being
-#       sent to the sender.
 
 BINARY="${1:?}"
 export SYSTEMD_LOG_LEVEL=info
 
-if [[ ! -x "$BINARY" ]]; then
+if ! [ -x "$BINARY" ]; then
     echo "$BINARY is not an executable"
     exit 1
 fi
 
 # output width
-if "$BINARY" --help | grep -v 'default:' | grep -E '.{80}.' >/dev/null; then
+if "$BINARY" --help | grep -v 'default:' | grep -qE '.{80}.'; then
```

Comment:
I'm not sure if sacrificing `set -o pipefail` and other "bashisms" is worth it here...

Reply:
Whereas I don't really see what `-o pipefail` gains us here.